### PR TITLE
[57392] Make phone number optional

### PR DESCRIPTION
**Pull request by tommasina-va**
When inputting contact information, make phone number optional to match the 526 paper form. 

vets-json-schema PR: https://github.com/department-of-veterans-affairs/vets-json-schema/pull/812

## Summary

- _This PR finishes the making of the phone number optional and includes some copy updates regarding phone number._

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
[department-of-veterans-affairs/va.gov-team#57392](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/57392)
- [merged] vets-json-schema PR: https://github.com/department-of-veterans-affairs/vets-json-schema/pull/806
- [new PR] vets-json-schema PR: https://github.com/department-of-veterans-affairs/vets-json-schema/pull/812
- [draft] vets-api PR: https://github.com/department-of-veterans-affairs/vets-api/pull/14407

## Testing done
- I tested locally that phone number is not required to continue through the 526 form
- I tested that behavior stays the same when phone number is filled in
- I tested on staging that without a phone number, the contact information card is in edit mode, and with a phone number it's in review mode. I also tested that it's not required to continue through the form, and that nothing on the Review and Submit page was affected/changed. 

## Screenshots

| Before  | After |
| ------------- | ------------- |
|image  | image  |
| image | image |

## What areas of the site does it impact?

526EZ

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by mchae-nava — APPROVED** [review]
Nice catches on the copy, and LGTM!

File: src/applications/disability-benefits/all-claims/content/claimExamsInfo.jsx [hunk 1]
```diff
@@ -18,6 +18,13 @@ export const claimExamsDescription = (
         schedule your exam. It’s important that you answer any calls you receive
         after you file a disability claim.
       </p>
+      <p>
+        You can go to your{' '}
+        <a href="/profile" target="_blank" rel="noreferrer">
+          VA.gov profile
+        </a>{' '}
```
Comment (christinec-fftc) [inline src/applications/disability-benefits/all-claims/content/claimExamsInfo.jsx:25]: For internal links, I learned we're supposed to open then in the same tab/window ([doc](https://design.va.gov/content-style-guide/links#linking-to-external-sites)). However, 526 has some odd behavior which prevents the user from easily getting back to the page they were on within the form. In that case, we should update the link text to include `(opens in new tab)`

Comment (tommasina-va) [reply src/applications/disability-benefits/all-claims/content/claimExamsInfo.jsx:25]: @christinec-fftc thanks so much! I'll do that.

**Review by ataker — APPROVED** [review]

File: src/applications/disability-benefits/all-claims/content/contactInformation.jsx [hunk 2]
```diff
@@ -12,7 +12,8 @@ const PhoneViewField = ({ formData: phoneNumber = '', name }) => {
   const phoneString = `${firstPhoneString}-${middlePhoneString}-${lastPhoneString}`;
   return (
     <p>
-      <strong>{name}</strong>: {phoneString}
+      <strong>{name}</strong>:{' '}
+      {phoneNumber.trim().length > 0 ? phoneString : ''}
```
Comment (Mottie) [inline src/applications/disability-benefits/all-claims/content/contactInformation.jsx:16]: I'd recommend using the `<va-telephone>` web component to render the phone number

**Review by Midge-dev — APPROVED** [review]
LGTM

**Pull request merged**